### ISSUE TITLE
Problems while training VIMO

### ISSUE THREAD

**Issue by liaowang0125** [yufu-wang/tram#24]
I tried to reproduce st_module and motion_module in my own code, but found that the rendering results were incorrect. If I only used st_module, the results were normal. I used batchsize=1, and extracted 16 frames per video. So I want to know:
1.if there are any details to pay attention to after adding these two modules
2.when will you release training data?
3.Is ther any loss use this? out['pred_rotmat_0'] = pred_rotmat_0
https://github.com/yufu-wang/tram/blob/main/lib/models/hmr_vimo.py#L115

**Comment by yufu-wang** [comment]
Thank you for trying to reproduce the training. I forget to mention this in the paper: you should apply Loss_rotmat both before and after the motion_module.
To make it more clear, apply all losses on the final results as usually. In addition, apply L_rotmat on the prediction (e.g. out['pred_rotmat_0']) before the motion_module. 
Hope this fixes it.

**Comment by liaowang0125** [comment]
Thanks,I will try

**Comment by yufu-wang** [comment]
I uploaded the training code. Please give it a try. Feel free to reopen the issue if you encouter problems.
